www/src/components/ecosystem/ecosystem-section.js

```diff
@@ -0,0 +1,147 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled from "react-emotion"
+import { Link } from "gatsby"
+
+import ArrowForwardIcon from "react-icons/lib/md/arrow-forward"
```

Comment:
not used

Reply:
we should provide some linting step or something to catch these early!

(tangential to this PR!)

Reply:
> we should provide some linting step or something to catch these early!

That would be great :+1: